<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" file="test_eos_exception_bytes.py" name="test_eos_exception_bytes.TestEosExceptionBytes-20230628194644" skipped="0" tests="1" time="0.001" timestamp="2023-06-28T19:46:44">
	<testcase classname="test_eos_exception_bytes.TestEosExceptionBytes" file="spec/python/test_eos_exception_bytes.py" line="7" name="test_eos_exception_bytes" time="0.001" timestamp="2023-06-28T19:46:44">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
